src/Client.php

```diff
@@ -172,33 +172,28 @@ protected function request($method, $path, array $options = [])
      */
     private function doRequest(RequestInterface $request, array $options)
     {
+        $exceptionMap = [
+            'InvalidQuery' => Exception\InvalidQueryException::class,
+            'AccessTokenInvalid' => Exception\AccessTokenInvalidException::class,
+            'NotFound' => Exception\NotFoundException::class,
+            'RateLimitExceeded' => Exception\RateLimitExceededException::class
+        ];
+
```

Comment:
Eventually we'll need to be able to override it to extend Exceptions for specific SDKs. I'd wanna wait to do so until we know what exactly we'll need.